lib/ansible/config/base.yml

```diff
@@ -571,6 +571,16 @@ DEFAULT_FORKS:
   ini:
   - {key: forks, section: defaults}
   type: integer
+DEFAULT_PROCESS_MODEL:
+  name: Sets the Ansible process model
+  default: "threading"
```

Comment:
I know the default was likely set to `threading` to get tests to run, but we need to remember to flip this.